### DISCUSSION TITLE
Configuring Metricbeat or Filebeat to Kibana issues

### DISCUSSION THREAD

**Discussion by jchurch93** [Security-Onion-Solutions/securityonion#4496] [Unsupported Versions]
I keep running into issues with either of these following the guides. 
Once I try to run 'metricbeat setup' I get the error: 

"Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to http://localhost:5601/api/status fails: <nil>. Response: {"statusCode":401,"error":"Unauthorized","message":"Unauthorized"}."

I've ready that securityonion directs everything through 443, so if I change the port to match that I get:
Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to https://xxxx:443/kibana/api/status fails: parsing kibana response: invalid character '<' looking for beginning of value. Response: <!doctype html> <!--   Copyright 2019 Jason Ertel (jertel). All rights reserved.   Copyright 2020-2021 Security Onion Solutions, LLC. All rights reserved.    This program is distributed under the terms of version 2 of the   GNU General Public License... (truncated).

I'm pretty much at a complete roadblock here, how have you configured the kibana section of your metricbeats.yml or filebeat.yml?

**Comment by weslambert** [comment]
The typical set up of Beat through Kibana will not really work as you expect at the moment.  If you just need to pick things up and send them with Filebeat, you should be able to install it locally on the source machine, then point it at the Security Onion manager (ensuring so-allow has been run to allow the port for Logstash/Beat), and point the Beat at $Security-Onion-Manager:5044 in the Filebeat configuration.

As far as Metricbeat is concerned, you can point it at the same port.  However, any templates, etc would need to be placed in `/opt/so/saltstack/local/salt/elasticsearch/templates/custom` , and any outputs for the Metricbeat data would need to reference the template.
